tests/src/test/scala/limits/ThrottleTests.scala

```diff
@@ -229,68 +229,6 @@ class ThrottleTests
       settleThrottles(alreadyWaited)
     }
   }
-
-  it should "throttle 'concurrent' activations of one action" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
```

Comment:
This test assumes the ShardingPoolBalancer is used.
The concurrency in the FPCscheduler differs from the ShardingPoolBalancer's one.

The sharding pool balancer counts the number of activations on the fly and throttles requests based on the concurrent invocation limit.
On the other hand, in the new scheduler, the concurrent limit implies the number of concurrent containers for a given action. Throttling works based on the processing power of existing containers.
When the maximum number of containers are being used and another action is invoked for the first time, it can't create more containers, it will reject the request with 429 too many requests.

Please refer to this comment: https://github.com/apache/openwhisk/blob/master/core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/FPCPoolBalancer.scala#L683

In this regard, I removed this test case.

Reply:
@style95 is there an alternate test that validates the behavior described int the comments?

Reply:
It's not the same, but there are two related tests.
The load balancer gets throttling flags from ETCD and rejects/accepts requests according to it.
So this test covers the behavior of the load balancer that decides based on the throttling flags.
https://github.com/apache/openwhisk/blob/master/tests/src/test/scala/org/apache/openwhisk/core/controller/test/FPCEntitlementTests.scala

And MemoryQueue populates these throttling flags with its state transition.
So this test covers if the memory queue sets the throttling key when it changes its state to a throttled state.
https://github.com/apache/openwhisk/blob/master/tests/src/test/scala/org/apache/openwhisk/core/scheduler/queue/test/MemoryQueueFlowTests.scala#L363

There are also some other tests related to throttling like
- [Throttling is enabled and disabled when it goes back to the Running state](https://github.com/apache/openwhisk/blob/master/tests/src/test/scala/org/apache/openwhisk/core/scheduler/queue/test/MemoryQueueFlowTests.scala#L479)